src/particles/profiles/GetInitialDensity.cpp

```diff
@@ -13,6 +13,7 @@ GetInitialDensity::GetInitialDensity (const std::string& name)
     amrex::ParmParse pp(name);
     std::string profile;
     getWithParser(pp, "density", m_density);
+    m_density = std::abs(m_density);
```

Comment:
I would clip instead of flip signs:
```suggestion
    m_density = m_density > 0._rt ? m_density : 0._rt;
```